wsclient/src/main/java/org/hpccsystems/ws/client/HPCCFileSprayClient.java

```diff
@@ -718,7 +722,74 @@ public boolean uploadFileLocalDropZone(File file) throws Exception
 
         return uploadFile(file, fetchLocalDropZones[0]);
     }
+    public boolean upLoadLandingZone(File uploadFile,DropZone dropZone) {
+  	
+    	String boundary = Utils.createBoundary();
+    	
+    	URLConnection fileUploadConnection = null;
+        URL fileUploadURL = null;
+        String FILESPRAYWSDLURI     = "/FileSpray";
+        String UPLOADURI           = FILESPRAYWSDLURI + "/UploadFile?upload_";
+        String uploadurlbuilder = UPLOADURI;
+        uploadurlbuilder += "&NetAddress=" + dropZone.getNetAddress();
+        uploadurlbuilder += "&Path=" + dropZone.getPath();
+        uploadurlbuilder += "&OS=" + (Utils.currentOSisLinux() ? "1" : "0");
+        try {
+			fileUploadURL = new URL(fsconn.getUrl() + uploadurlbuilder);
+			  fileUploadConnection = Connection.createConnection(fileUploadURL);
+			  fileUploadConnection = fileUploadURL.openConnection();
+	        	fileUploadConnection.setDoOutput(true);
+	        	fileUploadConnection.setRequestProperty("Content-Type", "multipart/form-data; boundary=" + boundary);
+	        	fileUploadConnection.setRequestProperty("Authorization", fsconn.getBasicAuthString());
+	        	OutputStream output = fileUploadConnection.getOutputStream();
+
+	        	Utils.startMulti(output, uploadFile.getName(), boundary, "");
+			InputStream input = new FileInputStream(uploadFile.getAbsolutePath());
+			
+			//using channels
+			ByteBuffer buffer = ByteBuffer.allocate(1000000);
```

Comment:
Is this a reasonable buffer size? Also we could make a multiple of the member var BUFFER_LENGTH